File: app/Models/StoreSetting.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class StoreSetting extends Model
{
    protected $fillable = [
        'user_id',
        'store_name',
        'store_address',
        'phone_number',
        'invoice_footer',
        'logo_path',
    ];

    public function user(): BelongsTo
    {
        return $this->belongsTo(User::class);
    }
}

File: mobile_app/lib/domain/entities/store_setting.dart
class StoreSetting {
  const StoreSetting({
    this.id,
    this.storeName = '',
    this.storeAddress = '',
    this.phoneNumber = '',
    this.invoiceFooter = '',
    this.logoUrl,
    this.updatedAt,
  });

  final int? id;
  final String storeName;
  final String storeAddress;
  final String phoneNumber;
  final String invoiceFooter;
  final String? logoUrl;
  final DateTime? updatedAt;
}
